onAssists was unhandled and cased into an error

nerdchandise opened the pull request:
Added unhandled `assist` Event

andrewda commented on lib/index.js:
```diff
@@ -209,6 +213,10 @@ Livescore.prototype._onSuicide = function(event) {
 };
 
+Livescore.prototype._onAssist = function(event) {
+    this.emit('assist', event);
```
See something like `onBombPlanted` for what should be emitted:

```
         this.emit('bombPlanted', {
             player: players[event.playerName]
         });
```

nerdchandise commented:
I changed the onAssist-Event so it emits player infos like the other events. 
Unfortunately the Assist is not included in the Kill-Event, so you have to emit both events. I added the killEventId, so you can assign the assist to a kill.
